robottelo/common/helpers.py

```diff
@@ -163,12 +164,13 @@ def valid_data_list():
     List of valid data for input testing.
     """
     return [
-        gen_string("alpha", 8),
-        gen_string("numeric", 8),
-        gen_string("alphanumeric", 8),
-        gen_string("utf8", 8),
-        gen_string("latin1", 8),
-        gen_string("html", 8)
+        gen_string('alphanumeric', randint(1, 255)),
+        gen_string('alpha', randint(1, 255)),
+        gen_string('cjk', randint(1, 85)),
+        gen_string('latin1', randint(1, 255)),
+        gen_string('numeric', randint(1, 255)),
+        gen_string('utf8', randint(1, 85)),
+        gen_string('html', randint(1, 85)),
     ]
```

Comment:
Nitpick: Couldn't we specify the max lengths as variables for an easier manipulation next time?